cmake/Abseil.cmake

```diff
@@ -0,0 +1,29 @@
+set(ABSL_ENABLE_INSTALL ON CACHE BOOL "" FORCE)
+if (NOT (DEFINED ABSEIL_PATH))
+  message("ABSEIL_PATH not provided. Downloading.")
+  set(ABSEIL_PREFIX "${benchmark_BINARY_DIR}/third_party/abseil")
+  configure_file(${benchmark_SOURCE_DIR}/cmake/Abseil.cmake.in ${ABSEIL_PREFIX}/CMakeLists.txt @ONLY)
```

Comment:
What's the motivation for using the template here rather than including `ExternalProject_Add` here directly?